03week/airplane/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="css/style.css" />
+
+<body>
+  <header>
+    <span>
+          <img src="img/logo.png" alt="" height=70px>
+        </span>
+    <nav>
+      <img src="img/home-icon.png" alt="">
+      <button2>Home</button2>
+      <img src="img/deal-icon.png" alt="">
+      <button1>Travel Deals</button1>
+      <img src="img/schedule-icon.png" alt="">
+      <button1>Flight Schedule</button1>
+      <img src="img/login-icon.png" alt="">
+      <button1>Register/Login</button1>
+    </nav>
+  </header>
+  <main>
+    <aside>
+      <div id="top">
+        <span id=blue>
```

Comment:
The span's that hold the 3 images are all scrunched together with no spacing, consider adding padding.